docs/src/explanations/interval-type.md

```diff
@@ -1,4 +1,11 @@
-# Intervals
+---
+layout: docs
+title:  "Interval Types"
+section: "chisel3"
+---
+
+# Interval Types
+
```

Comment:
It looks like this content was copied into the `experimental features` Appendix, so perhaps we should delete this instead? https://www.chisel-lang.org/chisel3/docs/appendix/experimental-features.html

Reply:
Sure. this is one of the pages that wasn't linked to, but I think I do need to fix a reference elsewhere. Will do.

Reply:
deleted and referred to the experimental features section